### PR TITLE
Fixing Master

### PR DESCRIPTION
**Pull request by roigcarlo**
**Description**
Seems this was failing. This is very unlekely to happen but it did. 
I have identified the mechanics from which chnges can bypass some CI checks and we will try to ensure it doesn't happen again.

@tteschemacher I am not enteirly sure this is the correct fix. Please feel free to change as needed.
@KratosMultiphysics/technical-committee Please approve this ASAP as otherwise nobody will be able to merge against master.

**Changelog**
- Fixing master errors in tests
- Fixing IGA Application

### CODE REVIEW

**Comment by tteschemacher** [comment]
Hey @roigcarlo ,

#9002 is fixing the merge things. Not sure about the shell error. I'll need to check that.

**Comment by roigcarlo** [comment]
A ok, didnt saw the PR. Thx

File: applications/IgaApplication/custom_elements/shell_5p_element.cpp [hunk 1]
```diff
@@ -199,8 +199,8 @@ namespace Kratos
         rKin.dtd1 = prod(rVar.P, rKin.dtd1);
         rKin.dtd2 = prod(rVar.P, rKin.dtd2);
 
-        rKin.metricChange[0] = inner_prod(rKin.A1, rKin.dud1) + 0.5 * norm_2_square(rKin.dud1);
-        rKin.metricChange[1] = inner_prod(rKin.A2, rKin.dud2) + 0.5 * norm_2_square(rKin.dud2);
+        rKin.metricChange[0] = inner_prod(rKin.A1, rKin.dud1) + 0.5 * std::sqrt(norm_2(rKin.dud1));
+        rKin.metricChange[1] = inner_prod(rKin.A2, rKin.dud2) + 0.5 * std::sqrt(norm_2(rKin.dud2));
```
Comment (tteschemacher) [inline applications/IgaApplication/custom_elements/shell_5p_element.cpp:203]: Hey, out of curiosity, why did you change this? is this failing?

Comment (roigcarlo) [reply applications/IgaApplication/custom_elements/shell_5p_element.cpp:203]: It was failing for me in win, but seems that is not giving problems anywhere else, so probably it was a problem in my side

**Pull request closed without merging**